servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -21,7 +22,32 @@ You can use these examples to configure requests to your services using the 3sca
 Pay particular attention to the `kind: handler` resource. You must update this with your 3scale credentials and the service ID of the API you want to manage.
 ====
 
-Modify the handler configuration with your 3scale configuration.
+. Modify the handler configuration with your 3scale configuration.
++
+[source,yaml]
+----
+  apiVersion: "config.istio.io/v1alpha2"
+  kind: handler
+  metadata:
+   name: threescale
+   namespace: istio-system
+  spec:
+   adapter: threescale
+   params:
+     service_id: "<SERVICE_ID>"
+     system_url: "https://<organization>-admin.3scale.net/"
+     access_token: "<ACCESS_TOKEN>"
+   connection:
+     address: "threescale-istio-adapter:3333"
+----
+
++
+. Create the required resources:
++
+----
+oc create -f istio/ -n istio-system
+----
+
 
 == Generating custom resources
 The adapter embeds a tool that allows generation of the `handler`, `instance`, and `rule` custom resources.
```

Comment:
Where is the tool embedded?  In 3Scale?  Or in OpenShift? @PhilipGough @dfennessy

Reply:
@JStickler its shipped with the adapter image so can be accessed locally if they pull the image or on OpenShift where the adapter is running

Reply:
I updated the language to clarify. See new content.